opencv/src/main/java9/module-info.java

```diff
@@ -0,0 +1,41 @@
+module org.bytedeco.javacpp.opencv {
```

Comment:
We'll also need to add `org.opencv` to the list.

Reply:
Right. Added.